jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -2202,7 +2202,7 @@
 					],
 					"request": {
 						"url": {
-							"raw": "{{url}}/v1.0/groups/{{HealthCheckGroup}}/apps/conf/{{HealthCheckWar}}/HEALTH-CHECK-4.0?hostName={{host}}",
+							"raw": "{{url}}/v1.0/groups/{{HealthCheckGroup}}/apps/conf/healthcheck-webapp-1.0.3.war/{{HealthCheckApp}}?hostName={{host}}",
```

Comment:
Also this test looks like its dependent on CTP health check being deployed. Can we change this to run on a postman deployed JVM?